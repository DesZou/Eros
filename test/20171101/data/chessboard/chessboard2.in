4
*ooo
oooo
oooo
*ooo
