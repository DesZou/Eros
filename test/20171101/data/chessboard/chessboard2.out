1
11519
4804
60
0
